tests/end2end/features/misc.feature

```diff
@@ -457,6 +457,48 @@ Feature: Various utility commands.
             - data/click_element.html
             - data/hello.txt (active)
 
+    Scenario: Clicking an element by CSS selector
+        When I open data/click_element.html
+        And I run :click-element css .clickable
+        Then the javascript message "click_element CSS selector" should be logged
+
+    Scenario: Clicking an element with non-unique filter
+        When I open data/click_element.html
+        And I run :click-element css span
+        Then the error "Multiple elements found matching css=span!" should be shown
+
+    Scenario: Clicking first element matching a selector
+        When I open data/click_element.html
+        And I run :click-element --select-first css span
```

Comment:
Should this have a "Then" of some sorts?

Reply:
Yes, thx.